src/endpoints/symbolicate.rs

```diff
@@ -44,6 +46,8 @@ struct SymbolicationRequestBody {
     pub stacktraces: Vec<RawStacktrace>,
     #[serde(default)]
     pub modules: Vec<RawObjectInfo>,
+    #[serde(default)]
+    pub options: Option<RequestOptions>,
```

Comment:
You can make this `RequestOptions` right away, since serde will then add the default for you.